### ISSUE TITLE
Search fails when no ranking data

### ISSUE THREAD

**Issue by mrghg** [openghg/openghg#131]
In ```demo``` branch:

I started to regenerate my local object store from scratch. Before I'd set any ranking, I ran:

```
results = search(species="ch4")
```

which returns:

```
ValueError: No ranking data set for the given search parameters. Please refine your search to include a specific site, species and inlet.
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-20-e0c0ad393821> in <module>
----> 1 results = search(species="ch4")

~/code/openghg/openghg/processing/_search.py in search(**kwargs)
    185 
    186     if not highest_ranked:
--> 187         raise ValueError(
    188             (
    189                 "No ranking data set for the given search parameters."

ValueError: No ranking data set for the given search parameters. Please refine your search to include a specific site, species and inlet.
```

Only works if I set ```skip_ranking=True```

**Comment by mrghg** [comment]
Actually, this also fails even if there is some ranking data...

I tried to set the BSD top inlet to be the preferred one:

```
r.set_rank(key="ch4_248m_picarro", rank=1)
```

But still get the same error message. Obviously this is also preventing ```get_obs_surface``` from running

**Comment by gareth-j** [comment]
I couldn't reproduce the not finding anything after ranking data has been set which is weird. I've added a check in so the `search` function skips the ranking code if there's no ranking data set.

**Comment by gareth-j** [comment]
See #141

**Comment by gareth-j** [comment]
Fixed with #141
